lib/compat/wordpress-6.3/class-gutenberg-rest-block-patterns-controller-6-3.php

```diff
@@ -138,6 +138,12 @@ public function get_item_schema() {
 					'type'        => 'string',
 					'readonly'    => true,
 					'context'     => array( 'view', 'edit', 'embed' ),
+					'enum'        => array(
+						'core',
+						'pattern-directory/core',
+						'pattern-directory/theme',
+						'pattern-directory/featured',
+					),
```

Comment:
We should add `plugin` and `theme` to this as mentioned in the docs I added yesterday.

Reply:
https://github.com/WordPress/gutenberg/pull/51672/commits/4021ff30e3c20170b6fed567c1d562ec4f39d6e8